templates/frontOffice/default/cart.html

```diff
@@ -121,7 +121,7 @@ <h3 class="name"><a href="{$PRODUCT_URL nofilter}">{$TITLE}</a></h3>
                                 </div>
                             </td>
                             <td class="subprice">
-                                <span class="price">{format_money number={$real_price * $QUANTITY} symbol={currency attr="symbol"}}</span>
+                                <span class="price">{format_money number={$real_price} symbol={currency attr="symbol"}}</span>
```

Comment:
i re-use $real_price, because it's no more used...
But your right, it's not clear, i'll change this